cubeseed_login/src/pages/_app.tsx

```diff
@@ -1,6 +1,11 @@
-import type { AppProps } from 'next/app'
-import "@/styles/globals.scss"
+import type { AppProps } from "next/app";
+import "@/styles/globals.scss";
+import { FarmContexProvider } from "@/context/context";
 
 export default function App({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />
+  return (
+    <FarmContexProvider>
```

Comment:
Why did you wrap the root component in this?

Reply:
i needed  to use a state in a different component ...so lifting it up was the only option for now .